lib/index.js

```diff
@@ -128,18 +134,25 @@ util.inherits(SpawnMocha, EventEmitter);
 
 var mochaStream = function mocha(opts) {
   opts = opts || {};
+  var files = [];
```

Comment:
No I am concerned that your code is reverting what gulp.src does. If someone is using gulp.src, it is because they want file by file effect.

It'll be better not to use stream at all in this case, just use node-glob, promises and the spawnMocha object. (That also show that in my current code, the error handling is at the wrong place.)